pages/transfer.vue

```diff
@@ -3,7 +3,7 @@
     <Loader v-model="isLoading" :status="status" />
     <nuxt-link
       v-if="$route.query.target"
-      :to="`/rmrk/u/${destinationAddress}`"
+      :to="`/${chainPrefix}/u/${correctAddress}`"
```

Comment:
what's correct address?

Reply:
> what's correct address?

correct address is either destination address or, in case user enters address for wrong chain, it gets transformed into correct address, i.e. KSM address on BSX won't be pointing to rmrk but to u/bsx/bsxaddress